x-pack/legacy/plugins/siem/public/components/stat_items/__snapshots__/index.test.tsx.snap

```diff
@@ -1053,6 +1053,8 @@ exports[`Stat Items Component rendering kpis with charts it renders the default
                                       },
                                     ]
                                   }
+                                  height={0}
+                                  width={0}
```

Comment:
The snapshot tests are returning `0` for `height` and `width`?

Looking below I don't see anything obvious to cause this.

Reply:
hm good question... digging deeper but not immediately apparent

Reply:
i merged in master and updated the snapshot again, seemed to knock out whatever this weirdness was.